$NetBSD: patch-ab,v 1.1 2007/03/17 15:34:13 rillig Exp $

g++4 complains:
constants.h:33: error: extra qualification 'gpspoint2::Constants::' on member 'Constants'

--- src/constants.h.orig	2003-04-14 18:23:32.000000000 +0200
+++ src/constants.h	2007-03-17 16:18:14.000000000 +0100
@@ -30,7 +30,7 @@ using namespace gpspoint2;
 class Constants
 {
    public:
-      Constants::Constants(void);
+      Constants(void);
 
    protected:
       static const g_byte
